Authentication/AuthenticationBootstrapper.swift

```diff
@@ -219,33 +235,29 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
 extension AuthenticationBootstrapper: LoginControllerTransitionDelegate {
     
     public func onSignup(controller: LoginController) {
```

Comment:
I know I told you to change it in other PR, but now I think that it would be more appropiate to name it `onSignupButtonPressed`. We should strive for "clarity on the place of use". Sorry for suggesting this change in thr other PR.

I know that because the function is in the LoginControllerTransitionDelegate, we can infer that it is from that protocol. But that is because we know what it means because we are designing the library. Furthermore, If you implement both delegates in the same extensions, you lose this "reference".
What do you think @guidomb @danielaRiesgo?